src/main/java/invtweaks/InvTweaksConfigManager.java

```diff
@@ -215,8 +217,9 @@ private boolean loadConfig() {
         }
     }
 
-    private boolean extractFile(ResourceLocation resource, File destination) {
-        try(InputStream input = mc.getResourceManager().getResource(resource).getInputStream()) {
+    private boolean extractFile(String resourcePath, File destination) {
+        ClassLoader classloader = Thread.currentThread().getContextClassLoader();
+        try (InputStream input = classloader.getResourceAsStream(resourcePath)) {
```

Comment:
Quick look at the code I bet it's a null issue -- there's one check still in the movement code. Probably a whole bunch of those randomly around, need to see if analyze finds any more.